nxcpy/components/_connected.pyx

```diff
@@ -0,0 +1,32 @@
+from libc.stdlib import malloc, free
+cimport _api.connected
+
+import networkx as nx
+
+__all__ = [
+        'connected_components',
+        'is_connected'
+]
+
+cdef enum Color:
+    WHITE = 0
+    GRAY = 1
+    BLACK = 2
+
```

Comment:
Cool.
If you need any help with this, I am free 
Waiting for the wrapper to be completed and for that first moment of satisfaction ! :smile: